### DISCUSSION TITLE
Seria interessante adicionar ao repo uma pasta com alguns dojos prontos para serem rodados?

### DISCUSSION THREAD

**Discussion by rkrocha** [lula42/42Labs#6] [Ideas]
Esses dojos teriam como assunto as linguagens e tecnologias mais comuns e/ou essenciais de se aprender durante o Labs. Por exemplo Python, JS, Docker, OOP, algum banco de dados, algumas ferramentas de teste, e coisas do tipo...

Cada um poderia ter uma documentação breve com instruções de como começar, links de referências, e um punhado de arquivos com uma "moldura" já formada para auxiliar no desenvolvimento do desafio.

Sendo ambicioso, alguns dos dojos com exigências mais complexas de preparação de ambiente (como potencialmente algo relacionado a bancos de dados) poderiam ter Dockerfiles e composes já prontos para configurar tudo com um único comando. Estes dojos fariam mais sentido caso tivessem como requisito algum conhecimento prévio sobre Docker, ou já ter completado o dojo de Docker.

**Comment by lula42** [comment]
Achei muito boa a ideia!

Criei uma Issue para quem quiser fazer: https://github.com/lula42/42Labs/issues/7

Vou deixar lá nos comments das Issues uns repos dos quais podemos pegar Dojos já prontos.
